features/composition-events.yml

```diff
@@ -0,0 +1,10 @@
+name: Composition events
+description: Composition Events are a method of adding text containing characters that might not be commonly available on keyboard, from emoji to logograms in Asian languages to input from speech recognition processors.
```

Comment:
Would you mind changing the descriptions for all of these to use the convention that we have for other event features?

> [one event] The `foo` event fires when…
> [multiple events] The `bar`, `baz`, and `qux` events fire when…
> [too many events to name] Mouse events, such as `click`, `mousedown`, or `mousemove`, fire when…

I'm guessing you've already familiarized yourself with these and it'll move a lot faster if you seed these conventionally first.